packages/notebook/src/actions.tsx

```diff
@@ -279,9 +279,13 @@ namespace NotebookActions {
     let state = Private.getState(widget);
     let cells = widget.model.cells;
     let widgets = widget.widgets;
+    let focus = -1;
     cells.beginCompoundOperation();
     for (let i = cells.length - 2; i > -1; i--) {
       if (widget.isSelectedOrActive(widgets[i])) {
+        if (focus == -1) {
```

Comment:
Here, and elsewhere, can you use `===` and `!==`?

Reply:
Yes, I will change this, thanks.